### DISCUSSION TITLE
Use corporate-friendly license

### DISCUSSION THREAD

**Discussion by ghost** [memgraph/memgraph#935] [Q&A] [answered]
I would be easier to get approval in a large company with a standard license, at least for a part of the code base such as the core memgraph. For example the Server Side Public License Mongo uses.

**Comment by gitbuda** [comment, marked as answer]
Hi @threedliteguy!

Interesting because we specifically picked the BSL license (with the conversation into the Apache2) because it should be easier to approve it inside large companies (in many cases, it should already be approved). Ofc, out of many other reasons. We should talk live to exchange more info + help somehow 😄
